static/css/query-builder.css

```diff
@@ -285,6 +285,10 @@ input.form-control {
     color: var(--text-color);
 }
 
+.ui-menu.ui-widget.ui-widget-content.ui-autocomplete.ui-front{
+    width: max-content!important;
```

Comment:
Looks good! You can add `min-width: 120px`, so for smaller text, it takes up the full space.
<img width="180" alt="image" src="https://github.com/siglens/siglens/assets/71771131/8531698a-4b2a-4c81-84ba-5acbe1d45547">